packages/@aws-cdk/aws-events-targets/lib/api-gateway.ts

```diff
@@ -0,0 +1,141 @@
+   *
+   * @default no querystring parameters
+   */
+  readonly queryStringParameters?: { [key: string]: (string) };
+
+  /**
+   * Message Group ID for messages sent to this queue.
+   *
+   * Required for FIFO queues, leave empty for regular queues.
+   *
+   * @default - no message group ID (regular queue)
+   */
+  readonly messageGroupId?: string;
+
+  /**
+   * The message to send to the queue.
+   *
+   * Must be a valid JSON text passed to the target queue.
+   *
+   * @default the entire EventBridge event
+   */
+  readonly message?: events.RuleTargetInput;
+
+  /**
+   * The role to assume before invoking the target
+   * (i.e., the pipeline) when the given rule is triggered.
+   *
+   * @default - a new role will be created
+   */
+  readonly eventRole?: iam.IRole;
+
+  /**
+   * The SQS queue to be used as deadLetterQueue.
+   * Check out the [considerations for using a dead-letter queue](https://docs.aws.amazon.com/eventbridge/latest/userguide/rule-dlq.html#dlq-considerations).
+   *
+   * The events not successfully delivered are automatically retried for a specified period of time,
+   * depending on the retry policy of the target.
+   * If an event is not delivered before all retry attempts are exhausted, it will be sent to the dead letter queue.
+   *
+   * @default - no dead-letter queue
+   */
+  readonly deadLetterQueue?: sqs.IQueue;
+}
+
+/**
+ * Use an API Gateway REST APIs as a target for Amazon EventBridge rules.
+ */
+export class ApiGateway implements events.IRuleTarget {
+
+  constructor(public readonly restApi: api.RestApi, private readonly props?: ApiGatewayProps) {
+  }
+
+  /**
+   * Returns a RuleTarget that can be used to trigger this API Gateway REST APIs
+   * as a result from an EventBridge event.
+   *
+   * @see https://docs.aws.amazon.com/eventbridge/latest/userguide/resource-based-policies-eventbridge.html#sqs-permissions
+   */
+  public bind(rule: events.IRule, _id?: string): events.RuleTargetConfig {
+    if (this.props?.deadLetterQueue) {
+      addToDeadLetterQueueResourcePolicy(rule, this.props.deadLetterQueue);
+    }
+
+    const restApiArn = this.restApi.arnForExecuteApi(
+      this.props?.method,
+      this.props?.path || '/',
+      this.props?.stage || this.restApi.deploymentStage.stageName,
+    );
+    return {
+      arn: restApiArn,
+      role: this.props?.eventRole || singletonEventRole(this.restApi, [new iam.PolicyStatement({
+        resources: [restApiArn],
+        actions: [
+          'execute-api:Invoke',
+          'execute-api:ManageConnections',
+        ],
+      })]),
+      deadLetterConfig: this.props?.deadLetterQueue && { arn: this.props.deadLetterQueue?.queueArn },
+      input: this.props?.message,
+      targetResource: this.restApi,
+      httpParameters: {
+        headerParameters: this.props?.headerParameters,
+        queryStringParameters: this.props?.queryStringParameters,
+        pathParameterValues: this.props?.pathParameterValues,
```

Comment:
Should we not validate `pathParameterValues.length` agains the number of `*` in the `path` ?

Reply:
I didn't know if it would be better to validate with CDK or wait for Cloudformation to handle the errors naturally at runtime. 
(If it is documented somewhere, I would appreciate it if you could let me know.)
I have implemented validation as you mentioned.